docs/core.adoc

```diff
@@ -10,22 +10,60 @@ It's main features are:
 
 == Installation
 
-Use the Cordova CLI to add the Core module as a plugin to your project:
+The `@aerogear/core` module is available via NPM:
 
 ----
-cordova plugin add @aerogearservices/core-cordova
+npm install @aerogear/core
 ----
 
-This will install the package from the NPM registry and properly link all native dependencies to the project's cordova Platforms.
+=== Additional steps for a Cordova project
+
+After node module installation via NPM use the Cordova CLI to add the additional native dependencies to your project:
+
+----
+cordova plugin add cordova-plugin-device
+cordova plugin add cordova-plugin-app-version
+----
+
+This will install the packages with these native dependencies from the NPM registry and properly link all native dependencies to the project's cordova Platforms.
 
 Refer to the link:./getting-started[Getting Started] guide for more information on installing the AeroGear SDK.
 
 == Usage
 
-After installation, the Core module will become available through the global `window.aerogear` object in your project.
+The Core module supports parsing of the configuration file generated by the mobile CLI:
```

Comment:
This has changed a little since yesterday, now the parser will be transparent for the user. It will only be required to import the JSON file and pass it to the constructor:
```js
import { MyService } from "@aerogear/my-module";
import appConfig from "../mobile-services.json";

const service = new MyService(appConfig);
```

Reply:
@josemigallas Let's propose some update here to simplify @paolobueno work

Reply:
:+1: I guess there's really not much 'usage' for the core module, I'll reword this as I split away the metrics part